src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -31,7 +32,9 @@ object DplaMapData {
 case class DplaMapData(
                          sourceResource: DplaSourceResource,
                          edmWebResource: EdmWebResource,
-                         oreAggregation: OreAggregation
+                         oreAggregation: OreAggregation,
+                         // Add id property at the root level (following ES schema)
+                         id: ExactlyOne[String]
```

Comment:
@mdellabitta I don't think so, `id` is a string but `@id` is a URI (at least according to what I see in ES) 

```
"_source": {
"isShownAt": "http://www.davidrumsey.com/luna/servlet/detail/RUMSEY~8~1~207512~3003388",
"object": "http://www.davidrumsey.com:8480/MediaManager/srvr?mediafile=/Size1/RUMSEY-8-NA/5025/0710_8.jpg",
"id": "0215aeb239de45b0dc789932607a1105",
"sourceResource": {},
"@id": "http://dp.la/api/items/0215aeb239de45b0dc789932607a1105",
"ingestType": "item",
"originalRecord": {}
}```